build/apis/githubapi-types.d.ts

```diff
@@ -78,16 +65,30 @@ export interface StatusEvent {
 
 // Github API -----------------------------------------------------------------
 // These provide the current bare minimum definitions for child Procbots working with them.
+export interface Comment {
+    body: string;
+    user: {
+        login: string;
+        type: string;
+    };
+    created_at: string;
+    updated_at: string;
```

Comment:
These are the types returned from the Github SDK API. Whilst it defines the object types we pass into it, it unfortunately doesn't do the same with return objects. Essentially this entire file is to make the returned objects from that API type safe (I am dreading turning this into a general purpose NPM module, as I'm going to spend a very boring day filling out a lot of these structures...).